### DISCUSSION TITLE
Single perimeter on bottom and top layer

### DISCUSSION THREAD

**Discussion by bigsubgit** [supermerill/SuperSlicer#1205] [General]
Hi everyone, I need to print with a single perimeter on bottom and top layer. For the top I set the parameter "only_one_peremeter_top". For the bottom I used a height range modifier, set from 0 to 0.2mm (just one layer of 0.2mm) with 1 perimeter, and range from 0.2mm to the top of the object, with 5 perimeter.
It works, but it makes only one bottom layer, instead of the 5 that I specified. 
Where I was wrong? Is it possible to do in another way?

https://user-images.githubusercontent.com/40430435/120110903-db514880-c16f-11eb-98c6-8997db838794.mp4

**Comment by supermerill** [comment]
you can set 100% rectilinear infill on layer 2-3

**Reply by bigsubgit** [reply]
Thanks @supermerill, it works.
